students/luftsmeerflier/lesson05/mailroom_3.py

```diff
@@ -30,6 +30,11 @@ def send_thank_you(first_name='', last_name=''):
 	else:
 		amount = input("How much did {} donate?\n".format(name))
 		date = input("When was the donation? YYYY-MM-DD\n")
+		try: 
+			print(amount > 0)
+		except:
+			print("That is an invalid donation amount")
```

Comment:
what exception do you anticipate to catch here?

please review course materials for exception handling: https://uwpce-pythoncert.github.io/PythonCertDevel/modules/Exceptions.html#bare-except

best practices is you want to avoid catching generic exceptions.


also, here would be a good place to convert amount to an integer and wrap that in try/except (with specific exception to catch).